extensions/default/src/CustomizeableContextMenu/ContextMenuItemsBuilder.test.js

```diff
@@ -0,0 +1,29 @@
+import ContextMenuItemsBuilder from "./ContextMenuItemsBuilder";
+
+const menus = [
+  {
+    id: 'one',
+    selector: ({ value }) => value === 'one',
+    items: [],
+  },
+  {
+    id: 'two',
+    selector: ({ value }) => value === 'two',
+    items: [],
+  },
+  {
+    id: 'default',
+    items: [],
+  },
+];
+
+const menuBuilder = new ContextMenuItemsBuilder();
+
+describe('ContextMenuItemsBuilder', () => {
+  test('findMenuDefault', () => {
+    expect(menuBuilder.findMenuDefault(menus, {})).toBe(menus[2]);
+    expect(menuBuilder.findMenuDefault(menus, { value: 'two' })).toBe(menus[1]);
```

Comment:
I must be missing something here, but shouldn't the second param to `findMenuDefault` have a `selectorProps` property? At least when I look at line 42 of ContextMenuItemsBuilder.ts.

Reply:
Yes, but selector props is really just a grab bag of things used for menu selection.